Tools/MiniBrowser/mac/WK2BrowserWindowController.m

```diff
@@ -532,8 +532,27 @@ - (void)didChangeSettings
     
     preferences._visibleDebugOverlayRegions = visibleOverlayRegions;
 
-    [_webView _setHeaderBannerHeight:[settings isSpaceReservedForBanners] ? testHeaderBannerHeight : 0];
-    [_webView _setFooterBannerHeight:[settings isSpaceReservedForBanners] ? testFooterBannerHeight : 0];
+    int headerBannerHeight = [settings isSpaceReservedForBanners] ? testHeaderBannerHeight : 0;
+    if (!headerBannerHeight)
+        [_webView _setHeaderBannerLayer:nil];
+    else {
+        CALayer *headerBannerLayer = [[CALayer alloc] init];
+        [headerBannerLayer setBounds:CGRectMake(0, 0, 0, headerBannerHeight)];
+        [headerBannerLayer setAnchorPoint:CGPointZero];
+        [headerBannerLayer setBackgroundColor:[NSColor colorWithSRGBRed:0xac / 255. green:0xdd / 255. blue:0xde / 255. alpha:1].CGColor];
```

Comment:
The mix of hex and decimal is confusing.

Reply:
Fixed